src/ducks/transactions/actions/AttachedDocsAction/Bill.jsx

```diff
@@ -0,0 +1,58 @@
+/* global __DEV__ */
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'cozy-ui/react'
+import Chip from 'cozy-ui/react/Chip'
+import FileOpener from 'ducks/transactions/FileOpener'
+import FileIcon from 'ducks/transactions/actions/AttachedDocsAction/FileIcon'
+import { Figure } from 'components/Figure'
+import styles from 'ducks/transactions/actions/AttachedDocsAction/Bill.styl'
+
+class DumbBill extends React.PureComponent {
+  static propTypes = {
+    bill: PropTypes.object.isRequired
```

Comment:
Nit: in cozy-ui we have started to have PropTypes representing doctypes. 

```
import {APP_TYPE } from 'cozy-ui/dist/propTypes' 
```

This could be an improvement to add the bill doctype and use it here.

Reply:
See https://github.com/cozy/cozy-ui/issues/1013. I will take the time to do it as soon as this feature is completed.